lib/CL/clCreateKernel.c

```diff
@@ -178,6 +178,7 @@ POname(clCreateKernel)(cl_program program,
       POCL_MEM_FREE(kernel->reqd_wg_size);
       POCL_MEM_FREE(kernel->dyn_arguments);
       POCL_MEM_FREE(kernel->arg_info);
+      POCL_MEM_FREE (kernel->name);
```

Comment:
Can you fix the argument spacing style for the others around it too?

Reply:
ok